lib/elixir/test/elixir/module/locals_tracker_test.exs

```diff
@@ -51,7 +51,11 @@ defmodule Module.LocalsTrackerTest do
     D.add_local(config[:ref], {:public, 1}, {:private, 1}, [line: 1, column: 1], false)
 
     undefined = D.collect_undefined_locals(config[:ref], @used)
-    assert undefined == [{{:public, 1}, [line: 1, column: 1], {:private, 1}, :undefined_function}]
+
+    assert undefined == [
+             {{:public, 1}, [span: {1, 8}, line: 1, column: 1], {:private, 1},
```

Comment:
I guess we can filter span out of here as well to keep just the position, but this should be inoffensive as well since it's just a meta

Reply:
This is private API, so we don't care. Having it here is fine. :)